src/__tests__/functional-tests/test-utils.test.tsx

```diff
@@ -7,9 +7,59 @@
 import React from 'react';
 import { render } from 'react-dom';
 import { render as renderTestingLibrary } from '@testing-library/react';
+import { pascalCase } from 'change-case';
 
 import Button from '../../../lib/components/button';
-import createWrapper from '../../../lib/components/test-utils/dom';
+import * as dom from '../../../lib/components/test-utils/dom';
+import * as selectors from '../../../lib/components/test-utils/selectors';
+import { getRequiredPropsForComponent } from '../required-props-for-components';
+import { getAllComponents, requireComponent } from '../utils';
+
+const createWrapperDom = dom.default;
+const createWrapperSelectors = selectors.default;
+
+const components = getAllComponents();
+const skipComponents = ['annotation-context', 'top-navigation', 'app-layout', 'modal'];
```

Comment:
Still don't know why `top-navigation`, `app-layout` and `modal` are failing. Will continue investigating.